.kitchen.yml

```diff
@@ -23,3 +23,8 @@ suites:
     run_list:
       - recipe[apt]
       - recipe[aurora::slave]
+
+  - name: client
+    run_list:
+      - recipe[apt]
```

Comment:
Well, this suite is tested on both debian and centos. On debian this is required to update the packages list, and this recipe doesn't do anything on centos.